packages/react-devtools-extensions/src/__tests__/parseHookNames-test.js

```diff
@@ -906,10 +910,9 @@ describe('parseHookNames worker', () => {
   beforeEach(() => {
     window.Worker = undefined;
 
-    workerizedParseSourceAndMetadataMock = jest.fn(() => {
-      console.log('mock fn');
-      return [];
-    });
+    workerizedParseSourceAndMetadataMock = jest.fn();
+
+    initFetchMock();
```

Comment:
just curious, what was the issue with the tests here?

meta: might be good to add some context to the commit summary itself, for when reviewing by commit

Reply:
We weren't mocking fetch for the tests at the end, so the `fetch` was failing which caused the test to fail. This didn't happen before because the fetch was done in the worker, which was _itself_ mocked out. Now that fetch is on the UI thread, we needed to mock it too (like we do with the above tests).